TheFourFifths/consus
# Checkout flow

### Summary

Provide a general description of the code changes in your pull request... were
there any bugs you had fixed? If so, mention them. If these bugs have open
GitHub issues, be sure to tag them here as well, to keep the conversation
linked together.

### Checklist


Finally, is it ready to be reviewed and merged: no :red_circle:

### How to Review

1. Look at `REST.md` to verify that all required endpoints are documented.
2. Verify that these endpoints have been implemented.
3. Verify that stores have been updated/implemented to accommodate the API requests.

### Links

- [TFF-55](https://msoese.atlassian.net/browse/TFF-55)
- [TFF-58](https://msoese.atlassian.net/browse/TFF-58)
- [API Doc](https://github.com/TheFourFifths/consus/blob/TFF-55-api/docs/REST.md)
